### DISCUSSION TITLE
Provide Flyte's project description for initialization

### DISCUSSION THREAD

**Discussion by robert-ulbrich-mercedes-benz** [flyteorg/flyte#5090] [RFC Incubator]
**Motivation**
Projects in Flyte can be structured according to departments or teams. It is nice to have the project description that can hold helpful information e.g. regarding contact person or team email addresses. The project description can currently only be set with the help of flytectl. This is not as nice to automate as if one could provide the project description e.g. in the Helm chart used for deployment.

**Functional**

Allow the initialization of a project with a project description.

**Comment by fg91** [comment]
Makes sense to me, would love to use as well.

**Comment by kumare3** [comment]
Sounds great. Lets do it!

**Comment by davidmirror-ops** [comment]
2024-04-25 Contributors meetup notes: no need for RFC, feel free to submit PR. Thanks
